docs/MEMORY-SETTINGS.md

```diff
@@ -220,6 +220,36 @@ Changing these parameters is recommended as a last resort, for example to debug
 an OOM kill due to a bloated `zedbox` process. Before changing the values,
 please read the [documentation](https://tip.golang.org/doc/gc-guide) carefully.
 
+## Forced execution of the Golang runtime garbage collector settings
+
+Setting a soft limit for Golang garbage collector may not be enough:
+the whole system may experience memory pressure due to other
+applications in the same of other memory cgroups. In such cases, GC
+can be forced from the memory pressure event handler. The following
+parameters are available for configuring this algorithm in order to
+minimize frequent GC calls:
+
+1. `gogc.forced.period.seconds` sets period of forced GC loop in seconds,
+    meaning that GC is called explicitly o more than once every 10 seconds.
+    Default value is 10 seconds.
+
+2. `gogc.forced.gained.mem.MiB` sets absolute amount of gained (allocated)
+   memory since last reclaim, meaning that GC will be called again only
+   if desired amount is gained. Default value is 50 MiB.
+
+3. `gogc.forced.gained.mem.percent` sets percent of last reclaimed
+   memory, which should be gained (allocated), meaning that GC will be
+   called again only if desired percentage of reclaimed (freed) memory
```

Comment:
Using the term "reclaimed" makes sense - perhaps the description can be augmented to say that it is what was reclaimed by the last forced GC (since there can be other reclaims happening).

But using the term and name "freed" for this is very confusing - the applications allocate and free memory and GC reclaims unreferenced memory.

Reply:
Well, GC anyway frees it from os perspective, because that what we want after all: to call unmap. I meant that. But I understand what you are saying, will fix